src/test/java/com/pubnub/contract/history/steps/ThenSteps.kt

```diff
@@ -0,0 +1,61 @@
+package com.pubnub.contract.history.steps
+
+import com.pubnub.api.MessageType
+import com.pubnub.api.models.consumer.history.PNFetchMessageItem
+import com.pubnub.api.models.consumer.history.PNFetchMessagesResult
+import com.pubnub.contract.history.state.FetchMessagesState
+import io.cucumber.java.en.Then
+import org.hamcrest.CoreMatchers.hasItems
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.empty
+
+class ThenSteps(
+    private val fetchMessagesState: FetchMessagesState
+) {
+
+    @Then("history response contains messages with {string} and {string} message types")
+    fun history_response_contains_messages_with_message_types(
+        messageTypeOfFirstMessage: String,
```

Comment:
I don't think there's anything about order of the messages itself. It's `firstMessageType` for sure because it's first in the step, but does it mean that the message will be first as well?

Reply:
So maybe `containsInAnyOrder` would be better here?

Reply:
So maybe "containsInAnyOrder" would be better